src/backend/api/Fusion.Resources.Domain/Queries/GetDepartments.cs

```diff
@@ -0,0 +1,48 @@
+﻿using Fusion.Resources.Database;
+using Fusion.Resources.Database.Entities;
+using Fusion.Resources.Domain.Models;
+using MediatR;
+using Microsoft.EntityFrameworkCore;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading;
+using System.Threading.Tasks;
+
+namespace Fusion.Resources.Domain
+{
+    public class GetDepartments : IRequest<IEnumerable<QueryDepartment>>
+    {
+        public string? DepartmentFilter { get; set; }
+        public string Sector { get; set; }
```

Comment:
I think i would make these properties private, and expose the usecase through builder methods, like mentioned earlier.. 

Just think of the next poor guy, to understand what these properties do... :)

Reply:
Sector should also be nullable